workflows/prognostic_c48_run/tests/test_regression.py

```diff
@@ -23,11 +23,88 @@
 LOG_PATH = "statistics.txt"
 RUNFILE_PATH = "runfile.py"
 CHUNKS_PATH = "chunks.yaml"
-DIAGNOSTICS = [
+DIAGNOSTICS_ML = [
```

Comment:
I suggest scoping this within `get_nudging_config` to reduce the amount of module level constants (especially since this is a mutable data structure).

Reply:
sounds good